### ISSUE TITLE
Need Member Functions For Arc Class

### ISSUE THREAD

**Issue by KevinKelly25** [smacademic/project-cgkm#32]
This Epic includes all the issues that involve adding all needed member function to the Arc Class. These issues are currently blocked by #22.

**Comment by KevinKelly25** [comment]
As we are going towards the BLoC design pattern most designed member functions are no longer needed. This epic will be closed until it is deemed that the arc class with need more member functions
